### ISSUE TITLE
Midterm Peer Review-as2739

### ISSUE THREAD

**Issue by salongaale** [JiaJiunn/orie4741_project#8]
The project focuses on examining election outcomes in swing counties across the country and what data can accurately predict voting patterns in them. This group used 50 fold cross validation, k nearest neighbors, decision trees, and a variety of regressions to try to accurately model the voting results for those swing counties. 
Positives for the project: 
-Good use of different methodologies to see how you can model your data
-Visualizations were useful in showing facts about the data and why you made certain conclusions
-Feature selection was not arbitrary and is well chosen

Improvements for the project:
-Lots of good models however when is it considered "good"? It seems you are aiming for the best accuracy however maybe look at the bias-variance tradeoff, the MSE, loss functions because accuracy alone might not be able to tell if your model tells a complete story
-How do you know if you are overfitting or underfitting? Include that in the final report as well
-Can you explain why you just dropped the missing data? 2% is alot for voting as it can determine an election outcome and you don't want a specific type of group to be consistently be missing data.
